Xamarin.Forms.Core/HandlerAttribute.cs

```diff
@@ -5,19 +5,60 @@ namespace Xamarin.Forms
 	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
 	public abstract class HandlerAttribute : Attribute
 	{
+		protected HandlerAttribute(Type handler, Type target, object[] filteredSdkVersionInformation) : this(handler, target)
```

Comment:
I added a ctor parameter because `UIUserInterfaceIdiom` was added as such. Instead of `object[]`, what would be a proper data type? The reason why I added it in the first place was breaking up `{isSupported, minimumSdkVersion, maximumSdkVersion}` didn't make sense because `isSupported` didn't make sense on its own. Would it be okay if there were three properties (perhaps `isSupported can be named `isFilteredSdkVersionSupported`)?